### PR TITLE
Adds forceJpg flag to convert live photos

### PR DESCRIPTION
**Pull request by migue1s**
Adds a `forceJPG` flag to force the lib to convert target images. This will be mostly helpful when images contain Live Images but only Still images are wanted.

How it works:
Calling `UIImageJPEGRepresentation` will force any images to return their JPG equivalent, even if they're Live Photos.

### CODE REVIEW

**Comment by rajputav** [comment]
nicee 💯

**Comment by SonnyBrooks** [comment]
💯

**Comment by ivpusic** [comment]
thanks a lot! could you rename `forceJPG` to `forceJpg`?

**Comment by ivpusic** [comment]
thanks a lot!

File: example/app.js [hunk 1]
```diff
@@ -134,6 +134,7 @@ export default class App extends Component {
       multiple: true,
       waitAnimationEnd: false,
       includeExif: true,
+      forgeJpg: true,
```
Comment (fubar) [inline example/app.js]: You have a typo here in "forge"

Comment (migue1s) [reply example/app.js]: Great catch, will do a follow-up PR

**Pull request merged**